src/Microsoft.Fast.Components.FluentUI.Generators/DesignTokenConstants.cs

```diff
@@ -0,0 +1,264 @@
+﻿namespace Microsoft.Fast.Components.FluentUI.Generators;
+
+/// <summary>
+/// This class contains all the design tokens as const <c ref="string" />s where the value represents 
+/// the <c ref="Type"> of the value of the design token
+/// </summary>
+public class DesignTokenConstants
+{
+    // General tokens
+    public const string direction = "FluentUI.Direction";
+    public const string disabledOpacity = "float?";
+
+    // Density tokens
+    public const string baseHeightMultiplier = "int?";
```

Comment:
Both of these multipliers will be going away in favor of the new density system, hopefully soon, but there are cases when these might have decimal values. I'd probably make them `float`.